.github/workflows/coverage.yml

```diff
@@ -0,0 +1,22 @@
+name: Coverage
+
+on: [pull_request, push]
+
+jobs:
+  coverage:
+    runs-on: ubuntu-latest
+    env:
+      CARGO_TERM_COLOR: always
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Rust
+        run: rustup toolchain install nightly-2022-05-09 --component llvm-tools-preview
+      - name: Install cargo-llvm-cov
+        uses: taiki-e/install-action@cargo-llvm-cov
+      - name: Generate code coverage
+        run: cargo llvm-cov --no-default-features --features debug --workspace --exclude "*tests*" --exclude "fil*" --lcov --output-path lcov.info
```

Comment:
This will re-run all the tests, right? We proabably just want to modify the existing workflows to generate code coverage.

Reply:
ok